examples/genetic_music/sketch.js

```diff
@@ -0,0 +1,240 @@
+/**
+ *  Example: Genetic music
+ */
+
+var synth;
+var sloop;
+
+var validNotes = [...Array(128).keys()];
```

Comment:
nice use of some ES6 features (spread operator and `.keys()`).

Some of these notes are really piercing, or inaudibly low. Maybe https://p5js.org/reference/#/p5/subset can help? I'd limit to those that appear on a piano or an organ (listed [here](http://www.inspiredacoustics.com/en/MIDI_note_numbers_and_center_frequencies))